main.js

```diff
@@ -0,0 +1,23 @@
+const handleBarsAppend = (data) => {
+  const source = $('#people-template').html();
+  const template = Handlebars.compile(source);
+  const newHTML = template(data);
+
+  $('body').append(newHTML);
+};
+
+const getData = (response) => {
```

Comment:
there's not a real need for this function and for the handleBarsAppend function - you could just insert the handleBarsAppend code inside the 'success' method in the ajax. It's good practice to create small functions, but in my opinion, in this case, it's unnecessary